build-and-test.ps1

```diff
@@ -23,19 +23,16 @@ $manifestRepo = $manifest.Repos[0]
 $activeOS = docker version -f "{{ .Server.Os }}"
 $builtTags = @()
 
-$buildFilter = $Filter
-if ($activeOS -eq "windows" -and $buildFilter -like "2.*") {
-    $buildFilter = "$buildFilter/nanoserver-sac2016/*"
-}
+$buildFilter = "$VersionFilter*/*/$OSFilter*"
```

Comment:
Can you log an issue to make the 1.* folder structure align with 2.* so that the `*` behind the Filters so that this logic can be changed to the following:
$buildFilter = '*'
if (-not [string]::IsNullOrEmpty($versionFilter))
{
    $buildFilter = "$versionFilter/$buildFilter"
}
if (-not [string]::IsNullOrEmpty($OsFilter))
{
    $buildFilter += "$buildFilter/$OsFilter/*"
} 

The `*` immediately after the two Filter references is not ideal.

Reply:
https://github.com/dotnet/dotnet-docker-nightly/issues/491